packages/server/tests/acceptance/data/conformity/utils/utils.ts

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+import { signTransaction } from '@hashgraph/json-rpc-relay/tests/helpers';
+import axios from 'axios';
+
+import { localNodeAccountPrivateKey, sendAccountAddress } from './constants';
+import { JsonRpcRequest, JsonRpcResponse, Transaction, TransactionResponse } from './interfaces';
+
+export async function getTransactionCount(relayUrl: string) {
+  const request = {
+    jsonrpc: '2.0',
+    id: 1,
+    method: 'eth_getTransactionCount',
+    params: [sendAccountAddress, 'latest'],
+  };
+
+  const response = await sendRequestToRelay(relayUrl, request as JsonRpcRequest, false);
+
+  return response.result;
+}
+
+export async function getLatestBlockHash(relayUrl: string) {
+  const request = {
+    jsonrpc: '2.0',
+    method: 'eth_getBlockByNumber',
+    params: ['latest', false],
+    id: 0,
+  };
+
+  const response = await sendRequestToRelay(relayUrl, request as JsonRpcRequest, false);
+
+  return response.result.hash;
+}
+
+export async function sendRequestToRelay(
+  relayUrl: string,
+  request: JsonRpcRequest,
+  needError: boolean,
+): Promise<JsonRpcResponse> {
+  try {
+    const response = await axios.post(relayUrl, request);
+    if (request.method === 'eth_sendRawTransaction') {
+      await global.relay.pollForValidTransactionReceipt(response.data.result);
+    }
+    return response.data;
+  } catch (error) {
+    console.error(error);
+    if (needError) {
+      return {
+        jsonrpc: '2.0',
+        id: request.id,
+        error: {
+          code: -32603,
+          message: error instanceof Error ? error.message : 'An unknown error occurred',
+        },
+      } as JsonRpcResponse;
```

Comment:
Why not return the actual error if such exists? e.g.

     if (error.response?.data) {
        return error.response.data;
      }